config/horizon.json

```diff
@@ -153,7 +153,7 @@
 		"themes/subscription-purchases": false,
 		"themes/text-search-lots": true,
 		"themes/theme-switch-persist-template": false,
-		"themes/assembler-first": true,
+		"themes/assembler-first": false,
```

Comment:
See https://github.com/Automattic/wp-calypso/pull/85019

Reply:
@miksansegundo do you mean you'll revert this after it's been approved and before merging to trunk?

Reply:
Should be, because the CfT for this feature is already running :)

Reply:
I realized that when I was AFK. I'll revert that today.